src/test/java/io/github/thebusybiscuit/slimefun4/implementation/items/TestSlimefunItem.java

```diff
@@ -110,7 +108,7 @@ void testRecipeType() {
     void testIsItem() {
         ItemStack item = new CustomItemStack(Material.BEACON, "&cItem Test");
         String id = "IS_ITEM_TEST";
-        SlimefunItem sfItem = TestUtilities.mockSlimefunItem(plugin, id, item);
+        SlimefunItem sfItem = TestUtilities.mockSlimefunItem(plugin, id, new CustomItemStack(Material.BEACON, "&cItem Test"));
```

Comment:
What's the point of this?

Reply:
I think its testing that `SlimefunItem#register` should change the underlying `ItemStack` in some way, thus `sfItem#isItem` should return false even if the two`ItemStack`s *were* the same when we started.